src/main/java/uk/gov/companieshouse/items/orders/api/controller/CertificateItemsController.java

```diff
@@ -70,4 +65,18 @@ public CertificateItemsController(final CreateItemRequestValidator validator,
         return ResponseEntity.status(HttpStatus.CREATED).body(createdCertificateItemDTO);
     }
 
+    @GetMapping("/certificates/{certificate_id}")
+    public ResponseEntity<Object> getCertificateItem(@PathVariable String certificate_id)
```

Comment:
I am wondering whether we should insist on an X-Request-ID being provided here as we do in the other request hander? You could argue that even where we are not using it in log messages here, we may later, and that effectively would change the contract slightly as the header would suddenly be required?

Reply:
Oh yes, true, I will add this in